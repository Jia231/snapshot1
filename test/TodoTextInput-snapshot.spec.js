import React from 'react'
import renderer from 'react-test-renderer'
import TodoTextInput from '../TodoTextInput'

/*test('snapshot TodoTextInput', () => {
    const component = renderer.create(<TodoTextInput editing onSave={() => { }} />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
})*/
//add editing prop will do an error since snapshot is built without it
test('snapshot TodoTextInput', () => {
    const component = renderer.create(<TodoTextInput editing onSave={() => { }} />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
})