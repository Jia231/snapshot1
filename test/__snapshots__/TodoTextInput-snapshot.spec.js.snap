exports[`test snapshot TodoTextInput 1`] = `
<input
  autoFocus="true"
  className="edit"
  onBlur={[Function]}
  onChange={[Function]}
  onKeyDown={[Function]}
  placeholder={undefined}
  type="text"
  value="" />
`;
